src/backend/strategies.rs

```diff
@@ -1078,7 +1061,50 @@ pub(crate) async fn run_strategy_task<'s>(
                                                             _ => {
                                                                 // nothing
                                                             }
-                                                        }                                                        
+                                                        }
+
+                                                        // If a data contract was registered, add it to
```

Comment:
![123](https://github.com/dashpay/rs-platform-explorer/assets/8050494/fe75dfd6-2ec8-4570-9399-dd936f741fd7)

Too much of levels there, I suggest to decompose the function into smaller blocks